Formula/php.rb

```diff
@@ -0,0 +1,324 @@
+    ENV.prepend "LDFLAGS", "-L#{Formula["tidy-html5"].opt_lib}"
+
+    system "make"
+    ENV.deparallelize
+    system "make", "install"
+
+    bin.install_symlink "phar.phar" => "phar"
+
+    config_path.install "php.ini-development" => "php.ini"
+    (config_path/"php-fpm.d").install "sapi/fpm/www.conf"
+    config_path.install "sapi/fpm/php-fpm.conf"
+    inreplace config_path/"php-fpm.conf", /^;?daemonize\s*=.+$/, "daemonize = no"
+
+    # patch PEAR so it installs extensions outside of the Cellar
+    Dir.chdir prefix do
+      pear_patch = Patch.create :p1, <<-EOS.undent
+        --- a/lib/php/PEAR/Builder.php	2017-09-07 23:46:07.000000000 -0500
+        +++ b/lib/php/PEAR/Builder.php	2017-09-07 23:47:17.000000000 -0500
+        @@ -405,7 +405,7 @@
+                 $prefix = exec($this->config->get('php_prefix')
+                                 . "php-config" .
+                                $this->config->get('php_suffix') . " --prefix");
+        -        $this->_harvestInstDir($prefix, $inst_dir . DIRECTORY_SEPARATOR . $prefix, $built_files);
+        +        $this->_harvestInstDir($this->config->get('ext_dir'), $inst_dir . DIRECTORY_SEPARATOR . $prefix, $built_files);
+                 chdir($old_cwd);
+                 return $built_files;
+             }
+        --- a/lib/php/PEAR/Command/Install.php	2017-09-07 23:45:56.000000000 -0500
+        +++ b/lib/php/PEAR/Command/Install.php	2017-09-07 23:42:22.000000000 -0500
+        @@ -379,7 +379,7 @@
+                     $newini = array();
+                 }
+                 foreach ($binaries as $binary) {
+        -            if ($ini['extension_dir']) {
+        +            if ($ini['extension_dir'] && $ini['extension_dir'] === $this->config->get('ext_dir')) {
+                         $binary = basename($binary);
+                     }
+                     $newini[] = $enable . '="' . $binary . '"' . (OS_UNIX ? "\\n" : "\\r\\n");
+      EOS
+      pear_patch.apply
+    end
+  end
+
+  def caveats
+    s = []
+
+    s << <<-EOS.undent
+      To enable PHP in Apache add the following to httpd.conf and restart Apache:
+          LoadModule php7_module #{HOMEBREW_PREFIX}/lib/httpd/modules/libphp7.so
+
+          <FilesMatch \.php$>
+              SetHandler application/x-httpd-php
+          </FilesMatch>
+
+      Finally, check DirectoryIndex includes index.php
+          DirectoryIndex index.php index.html
+    EOS
+
+    s << <<-EOS.undent
+      The php.ini and php-fpm.ini file can be found in:
+          #{etc}/php/#{php_version}/
+    EOS
+
+    s.join "\n"
+  end
+
+  def post_install
+    (var/"log").mkpath
+    touch var/"log/php-fpm.log"
+
+    chmod 0755, lib/"php/.channels"
+    chmod 0755, lib/"php/.channels/.alias"
+    chmod 0644, (Dir.glob(lib/"php/.channels/**/*", File::FNM_DOTMATCH).reject { |a| a =~ %r{\/\.{1,2}$} || File.directory?(a) })
+    chmod 0644, %w[php/.depdblock php/.filemap php/.depdb php/.lock]
```

Comment:
These are the wrong paths. I cleaned it up at bit at https://github.com/kabel/homebrew-core/blob/a910b289ba94cf837e4a6c3ad5c89bf51c17ef5e/Formula/php.rb#L240-L255